modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/AbstractMvPartitionStorageTest.java

```diff
@@ -1269,8 +1269,22 @@ public void scanCursorHasNextReturnsFalseEachTimeAfterExhaustion(ScanTimestampPr
 
     @ParameterizedTest
     @EnumSource(ScanTimestampProvider.class)
-    public void scanDoesNotSeeTombstonesWhenTombstoneIsNotCommitted(ScanTimestampProvider tsProvider) throws Exception {
-        testScanDoesNotSeeTombstones(tsProvider, false);
+    public void scanSeesTombstonesWhenTombstoneIsNotCommitted(ScanTimestampProvider tsProvider) throws Exception {
```

Comment:
Let's inline `testScanDoesNotSeeTombstones()`, it looks like it is now being used only in one place